test/runtests.jl

```diff
@@ -32,6 +32,17 @@ const nw = parsed_args["parallel"]
 const viscoacoustic = parsed_args["viscoacoustic"]
 
 
+# Utility macro to run block of code with a single omp threa
+macro single_threaded(expr)
+    return quote
+        nthreads = ENV["OMP_NUM_THREADS"]
+        ENV["OMP_NUM_THREADS"] = "1"
+        local val = $(esc(expr))
+        ENV["OMP_NUM_THREADS"] = nthreads
```

Comment:
still there could be multiple threads if `ENV["OMP_NUM_THREADS"]` is initially not one?

Reply:
Zero idea what you're talking about. This is running an expression with a single thread and restoring env afterwards.

Reply:
Ohhh now I get it